src/utils/utils.py

```diff
@@ -0,0 +1,111 @@
+import os
+from uuid import UUID
+from flask import make_response
+from datetime import datetime
+import json
+from enum import Enum
+
+
+def ValidateUniqueValues(data):
+    """
+    Verify unique fields existence and required strings length.
+
+    :param dict data: Posted JSON data
+    :return: Validation result
+    """
+    validity = True
+    ids = []
+    vehicles = data["vehicles"]
```

Comment:
In your implementation you only use your validation for this already schema validated data; so that's fine.
If you were reusing the functionality you should consider some more things.
I would prefer to use the `get` method of `dict`, so no possible `KeyError` is raised. I think it is a good practice in general even though python tends to embrace EAFP.